engine/src/main/java/com/sandboni/core/engine/Arguments.java

```diff
@@ -18,7 +18,7 @@
     private String repository;
     private String outputFormat;
     private String trackingSort;
-    private boolean runSelectiveMode;
+    private boolean runOnConfigChange;
```

Comment:
Run on Config change is very specific and we are not using this flag to only validate config changes. We also use it to check if other non java files were changed, this could be Java Script, Groovy, Scala, etc, changes. Can we find a different name that covers these scenarios?

Reply:
I see, in that case "runSelectiveMode" is more appropriate for the core.